helm/aws-node-termination-handler/Chart.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: v1
-appVersion: 1.17.2
+appVersion: 1.22.1
 description: Deploy components for the aws-node-termination-handler app
-name: aws-node-termination-handler-app
-version: "[[ .Version ]]"
+name: aws-node-termination-handler
+version: [[ .Version ]]
```

Comment:
This needs to be removed , that _template_ is not replaced anymore, since `abs` doesn't use it.